lib/orderbook/OrderBook.ts

```diff
@@ -354,15 +355,28 @@ class OrderBook extends EventEmitter {
       };
     }
 
-    if (!this.nosanitychecks) {
-      // check if sufficient outbound channel capacity exists
-      const { outboundCurrency, outboundAmount } = Swaps.calculateInboundOutboundAmounts(order.quantity, order.price, order.isBuy, order.pairId);
-      const swapClient = this.swaps.swapClientManager.get(outboundCurrency);
-      if (!swapClient) {
+    if (this.swaps && !this.nosanitychecks) {
+      const {
+        outboundCurrency,
+        inboundCurrency,
+        outboundAmount,
+        } = Swaps.calculateInboundOutboundAmounts(order.quantity, order.price, order.isBuy, order.pairId);
```

Comment:
minor nit but there's extra indent here relative to how it's written elsewhere in the code, the `}` should line up with `const`.